unit_tests/test_aggregator.py

```diff
@@ -0,0 +1,415 @@
+            aggregator_explicit.aggregator_uuid = test_agg["uuid"]
+            aggregator_explicit.start_websocket_connection()
+            aggregator_explicit.execute_batch_commands()
+            mocked_method.assert_called_with(*args_)
+
+    @staticmethod
+    def test_execute_batch_commands__post_request_not_posted(aggregator_explicit):
+        with patch("gsy_e_sdk.aggregator.Aggregator._post_request",
+                   return_value=["test_transaction_id", False]):
+            assert aggregator_explicit.execute_batch_commands() is None
+
+    @staticmethod
+    def test_execute_batch_commands__clear_command_buffer(aggregator_explicit):
+        with patch("gsy_e_sdk.aggregator.ClientCommandBuffer.clear") as mocked_method:
+            aggregator_explicit.start_websocket_connection()
+            aggregator_explicit.execute_batch_commands()
+            mocked_method.assert_called_once()
+
+    @staticmethod
+    def test_execute_batch_commands__wait_for_command_response_called(aggregator_explicit):
+        with patch.object(AggregatorWebsocketMessageReceiver, "wait_for_command_response",
+                          return_value=test_response) as mocked_method:
+            aggregator_explicit.start_websocket_connection()
+            aggregator_explicit.execute_batch_commands()
+            mocked_method.assert_called_once()
+
+    @staticmethod
+    def test_execute_batch_commands__return_response(aggregator_explicit):
+        with patch.object(AggregatorWebsocketMessageReceiver, "wait_for_command_response",
+                          return_value=test_response):
+            aggregator_explicit.start_websocket_connection()
+            response = aggregator_explicit.execute_batch_commands()
+            assert response == test_response
+
+    @staticmethod
+    def test_execute_batch_commands__log_bid__offer_confirmation_called(aggregator_explicit):
+        with patch("gsy_e_sdk.aggregator.log_bid_offer_confirmation") as mocked_method:
+            aggregator_explicit.start_websocket_connection()
+            aggregator_explicit.execute_batch_commands()
+            mocked_method.assert_called()
+
+    @staticmethod
+    def test_execute_batch_commands__log_deleted_bid_offer_confirmation_called(
+            aggregator_explicit):
+        with patch("gsy_e_sdk.aggregator.log_deleted_bid_offer_confirmation") \
+                as mocked_method:
+            aggregator_explicit.start_websocket_connection()
+            aggregator_explicit.execute_batch_commands()
+            mocked_method.assert_called()
+
+
+@pytest.mark.parametrize("aggregator_specs", [test_agg, None])
+def test_list_aggregators(aggregator_specs, aggregator_explicit):
+    with patch("gsy_e_sdk.aggregator.blocking_get_request", return_value=aggregator_specs):
+        aggs_list = aggregator_explicit.list_aggregators()
+
+        if aggregator_specs is not None:
+            assert aggs_list == test_agg
```

Comment:
shouldn't this be
```suggestion
            assert aggs_list == [test_agg]
```
?